daemon/containerd/image_inspect.go

```diff
@@ -14,6 +14,7 @@ import (
 	imagetypes "github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/api/types/storage"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"golang.org/x/exp/maps"
```

Comment:
I think this is now in stdlib? (we should avoid depending on `golang.org/x/exp`)

Reply:
oh, but _if_ we use `stdlib`, we likely need to add a `//go:build go1.21` built-tag (but check with https://github.com/moby/moby/pull/46941)

Reply:
`maps.Keys` was added in go 1.23. We're still on 1.22 in our `vendor.mod`.

Reply:
Added a TODO comment

Reply:
Could we implement a minimal helper for this? It feels like a very minimal enhancement for it to introduce that as a direct dependency (I'm pretty sure we have various other places where we have a small utility to get all keys from a map).

Reply:
Actually, as we're already looping and already keeping track of duplicates, I wonder if we need it at all 🤔. here's a quick attempt without;

- https://github.com/vvoland/moby/pull/14

Reply:
Changed to `sliceutil.Dedup` as suggested by Albin.

Reply:
Ah, nice! Yes, that works